src/components/Tags.tsx

```diff
@@ -1,17 +1,22 @@
+import { differenceBy, isEqual, uniq } from 'lodash';
 import * as React from 'react';
-import { MBTTAGS } from './TagSuggestions';
-
 //Dependencies for drag n drop tags with suggestions
 //npm install --save react-tag-input
 //npm install --save react-dnd@5.0.0
 //npm install --save react-dnd-html5-backend@3.0.2
 import { WithContext as ReactTags } from 'react-tag-input';
-import { stringify, v4 as uuid } from 'uuid';
+import { assertModelArrayType, Segment, SegmentTag } from 'types';
+import { MBTTAGS } from './TagSuggestions';
+
+type ReactTag = {
```

Comment:
Note that for types that will be used outside of this component should be defined here:

https://github.com/cusac/mbt.guide/tree/master/src/types

However since we are only using this type within this component defining it here is more efficient.